sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -312,6 +315,9 @@
    */
   private final @NotNull List<String> contextTags = new CopyOnWriteArrayList<>();
 
+  /** Whether to send client reports containing information about number of dropped events. */
+  private @NotNull boolean sendClientReports = true;
```

Comment:
```suggestion
  private boolean sendClientReports = true;
```

You don't need this annotation if its a primitive type.